src/processors/transform.rs

```diff
@@ -0,0 +1,314 @@
+
+extern crate cgmath;
+// extern crate test;
+
+use self::cgmath::{Quaternion, Vector3, Matrix3, Matrix4};
+
+use ecs::{Join, Component, NullStorage, VecStorage, Entity, RunArg, Processor};
+use context::Context;
+use std::sync::{Mutex, Arc};
+use std::collections::HashMap;
+
+/// Local position and rotation from parent.
+#[derive(Debug, Clone)]
+pub struct LocalTransform {
+    pos: [f32; 3], // translation vector
+    rot: [f32; 4], // rotation (radians) vector
```

Comment:
This comment is slightly confusing. Could you change it to `rotation quaternion` (or sth similiar) and specify the component layout (vec - scalar)?